internal/pkg/crawl/worker.go

```diff
@@ -134,7 +133,7 @@ func (w *Worker) Run() {
 // unsafeCapture is named like so because it should only be called when the worker is locked
 func (w *Worker) unsafeCapture(item *queue.Item) {
 	if item == nil {
-		return
+		panic("Item is nil")
```

Comment:
Do we really want to panic? Maybe a log + return would be wiser. (even though this should never happen, right)

btw, error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation in Go

Reply:
oh, I accidentally put the change in this commit :(

Yeah, that will never happen.